go/vt/wrangler/keyspace.go

```diff
@@ -974,15 +970,15 @@ func (wr *Wrangler) waitForDrainInCell(ctx context.Context, cell, keyspace, shar
 	startTime := time.Now()
 	for {
 		// map key: tablet uid
-		drainedHealthyTablets := make(map[uint32]*discovery.LegacyTabletStats)
-		notDrainedHealtyTablets := make(map[uint32]*discovery.LegacyTabletStats)
+		drainedHealthyTablets := make(map[uint32]*discovery.TabletHealth)
+		notDrainedHealtyTablets := make(map[uint32]*discovery.TabletHealth)
 
-		healthyTablets := tsc.GetHealthyTabletStats(keyspace, shard, servedType)
```

Comment:
Why `GetTabletStats` instead of `GetHealthy..`? The new healthcheck also has a `GetHealthy..` func.